controls/datagrid/row-details/row-details-template.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Row Details Template
+page_title: .NET MAUI DataGrid Documentation - Row Details Template
+description: Learn how to present additional information about DataGrid row by using row details functionality.
+position: 
+slug: datagrid-row-details-template
+---
+
+# .NET MAUI DataGrid Row Details Template
+
+The Telerik UI for .NET MAUI DataGrid control define a `RowDetailsTemplate` property (type `DataTemplate`). The property is used for displaying row details. 
+
+
+## Example 
```

Comment:
To be added later? 
You don't need a dedicated section for the example. Adding a sentence that introduces the example is the better approach in such short articles.